Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -1065,6 +1065,7 @@ - (void)viewDidAppear:(BOOL)animated
     [self startReadTimer];
     [self updateNavigationBarSubtitleLabel];
     [self updateBackButtonUnreadCount];
+    [self autoLoadMoreIfNecessary];
```

Comment:
This fix is already in v2.27, but we'll want it in v2.26 too.

Reply:
For perf we're deliberately only loading a very small window - just enough to fill the screen (though our approach is pretty naive, we could probably load even less if we accounted for device size and message size).

What's wrong with waiting until we scroll to load more?

Reply:
> What's wrong with waiting until we scroll to load more?

There are common scenarios (e.g. more than a load page of unread messages) where we'll load messages up to the unread indicator and then auto-scroll so that the unread-indicator is near the top of the screen.  In these scenarios, the "load more messages" indicator is often visible above the unread indicator.  The rule is: auto-load more messages if we're "near" the "load more messages" indicator so that users don't see it.  That's a courtesy to the user so that the conversation feels infinite and the user doesn't notice loading.  There's another reason too: it avoids a nasty issue wherein it's hard to preserve scroll state if auto-loading occurs while the user is scrolling.

Anyway, performing `autoLoadMoreIfNecessary` here shouldn't do anything most of the time, since most of the time we're not "near enough" to the "load more messages" indicator - and when it does load more, it's for a good reason.